tests/data_context/datasource/test_data_context_datasource_runtime_data_connector_sparkdf_execution_engine.py

```diff
@@ -139,31 +139,6 @@ def test_batch_data_get_batch_successful_specification_sparkdf_engine_named_asse
     assert batch_2.batch_definition.batch_identifiers == batch_identifiers
 
 
-def test_batch_data_get_batch_ambiguous_parameter_sparkdf_engine(
```

Comment:
@donaldheppner I do not intuitively see why these tests are being removed.  Could you please help with a justification?  Thank you.

Reply:
These tests check the RuntimeBatchRequest constructor for the same logic that is tested other places and in `test_batch.py` -- the actual assertions here have nothing to do with Spark.